target/linux/mediatek/dts/mt7986a-asus-tuf-ax6000.dts

```diff
@@ -0,0 +1,382 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
```

Comment:
Judging from pictures on Google photos, this is a different base board than the ax4200. I've had a look, since my ax4200 came with an unpopulated area for a second 2.5GE PHY.

Reply:
base board looks different, that's true. What do you suggest?

Reply:
No suggestion, just an observation. I've thought this comment was separate from the review.

Reply:
> No suggestion, just an observation. I've thought this comment was separate from the review.

Ok. This router have a different case and main board than ax4200. I have both of them and I recommend to buy the ax6000 version.